
This adds capability to list episodes when right arrow is pressed on recording list


diff --speed-large-files --no-dereference --minimal -Naur Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/recordings-ui.xml Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/recordings-ui.xml
--- Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/recordings-ui.xml	2022-08-14 11:45:50.859967820 +0200
+++ Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/recordings-ui.xml	2022-08-14 11:31:26.522446464 +0200
@@ -33,6 +33,7 @@
             <wrapstyle>items</wrapstyle>
             <arrange>fixed</arrange>
             <showarrow>no</showarrow>
+            <triggerevent context="TV Frontend" action="LISTRECORDEDEPISODES">RIGHT</triggerevent>
             <statetype name="buttonitem">
                 <area>0,0,100%,50</area>
                 <state name="active">
